src/styles.js

```diff
@@ -93,6 +142,7 @@ module.exports = (theme) => ({
         h1: {
           color: theme('colors.gray.900', defaultTheme.colors.gray[900]),
           fontWeight: '800',
+          marginTop: '0',
```

Comment:
h1.marginTop can be moved to the first array item of the DEFAULT variant, because this rule was the only rule that differed the 2nd DEFAULT style object from the rest of the variant declarations